e2e/interchaintestv8/.golangci.yml

```diff
@@ -51,6 +51,10 @@ issues:
   max-same-issues: 10000
 
 linters-settings:
+  gosec:
```

Comment:
The new version of golangci-lint has a really aggressive check for casting of ints, which we have to do a lot with heights (and there is no "safe" way that will shut it up)